src/vellum/workflows/utils/validate.py

```diff
@@ -16,6 +16,10 @@ def __init__(self, declared_type: type, target_type: type) -> None:
 
 
 def validate_target_type(declared_type: type, target_type: type) -> None:
+    # If either type is Any, validation always passes
+    if declared_type is Any or target_type is Any:
```

Comment:
DevinAI - we should be able to remove the `is Any` check in the final output node now